test/ui/defaultHtmlStepUi.spec.js

```diff
@@ -15,6 +15,16 @@ describe('Sequencer step HTML', function() {
   };
   // options = JSON.parse(options)
 
+  beforeAll(() => {
+    const { JSDOM } = require('jsdom');
+    var { window } = new JSDOM();
+
+    const { document } = window;
+    global.window = window;
+    global.document = document;
+    const $ = global.jQuery = global.$ = require('jquery');
+  });
+
```

Comment:
Why do we have these imports here ? we are already requiring jsdom at the top! any reason ? I have been stuck with jasmine-jquery lemme know why it has been done! thanks

Reply:
It was not working for some reason.

Reply:
Worked after doing this

Reply:
You have to initialize the DOM in the beforeAll

Reply:
idk i tested and it worked!